### DISCUSSION TITLE
Display Port Working on 9470m

### DISCUSSION THREAD

**Discussion by paasoares** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#79] [Show and tell]
Hi Chris,
Thanks for your great work !
Followed your guide and everything is working perfectly (latest version of Big Sur 11.6)
Just to let you know that with some little changes on config.plist, External Monitor via Display Port is working correctly.

Here is what i've changed :
![Screenshot 2021-09-21 at 00 31 20](https://user-images.githubusercontent.com/1421196/134090414-496d1819-7589-470b-b4a4-582754b1b890.png)

and this is the result : 
![Screenshot 2021-09-21 at 00 28 52](https://user-images.githubusercontent.com/1421196/134090455-dcb77edc-0314-4071-83dd-30b2b08495cf.png)

Cheers

**Comment by chris1111** [comment]
Ok I will make change for the High Screen

**Comment by chris1111** [comment]
Done
![Screen Shot ](https://user-images.githubusercontent.com/6248794/134236143-ecfb862b-d3db-4b7e-b240-fe6a4b69bc64.png)
